### PR TITLE
Initial firmware for Retro75

### PR DESCRIPTION
**Pull request by zvecr**
## Description

Initial firmware for Retro75.

Tagging @PheonixStarr as pcb creator.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/retro_75/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Retro75
+
+![Retro75](https://i.imgur.com/spS7PHo.jpg)
+
+A 75% ARM STM32F072 keyboard, with WS2812 LEDs.
+
+Keyboard Maintainer: [zvecr](https://github.com/zvecr)
```
Comment (fauxpark) [inline keyboards/retro_75/readme.md]:
```suggestion
* Keyboard Maintainer: [zvecr](https://github.com/zvecr)
```

File: keyboards/retro_75/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Retro75
+
+![Retro75](https://i.imgur.com/spS7PHo.jpg)
+
+A 75% ARM STM32F072 keyboard, with WS2812 LEDs.
+
+Keyboard Maintainer: [zvecr](https://github.com/zvecr)  
+Hardware Supported: Retro75 PCB
```
Comment (fauxpark) [inline keyboards/retro_75/readme.md]:
```suggestion
* Hardware Supported: Retro75 PCB
```

File: keyboards/retro_75/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Retro75
+
+![Retro75](https://i.imgur.com/spS7PHo.jpg)
+
+A 75% ARM STM32F072 keyboard, with WS2812 LEDs.
+
+Keyboard Maintainer: [zvecr](https://github.com/zvecr)  
+Hardware Supported: Retro75 PCB  
+Hardware Availability: Private Group-Buy
```
Comment (fauxpark) [inline keyboards/retro_75/readme.md]:
```suggestion
* Hardware Availability: Private Group-Buy
```

File: keyboards/retro_75/readme.md (hunk 1 above)
Comment (zvecr) [reply keyboards/retro_75/readme.md]: Shoot, forgot about that in the template cleanup.

**Pull request merged**